core/node/rpc/notification_test.go

```diff
@@ -1564,4 +1564,28 @@ func spaceChannelSettings(
 
 	test.req.Equal(request3.Msg.ChannelId, channel2.ChannelId)
 	test.req.Equal(request3.Msg.Value, channel2.Value)
+
+	request5 := connect.NewRequest(&SetSpaceChannelSettingsRequest{
+		ChannelId: channel2ID[:],
+		SpaceId:   test.spaceID[:],
+		Value:     SpaceChannelSettingValue_SPACE_CHANNEL_SETTING_UNSPECIFIED,
+	})
+	authorize(ctx, test.req, test.authClient, user, request5)
+
+	_, err = test.notificationClient.SetSpaceChannelSettings(ctx, request5)
+	test.req.NoError(err, "SetSpaceChannelSettings failed")
+
+	authorize(ctx, test.req, test.authClient, user, request4)
+
+	_, err = test.notificationClient.SetSpaceSettings(ctx, request4)
+	test.req.NoError(err, "SetSpaceSettings failed")
+
+	// ensure that the settings are correct applied
+	settingsResp, err = test.notificationClient.GetSettings(ctx, request1)
+	test.req.NoError(err, "GetSettings failed")
+
+	settings = settingsResp.Msg
+
+	space = settings.GetSpace()[0]
+	test.req.Equal(1, len(space.Channels))
```

Comment:
Nitpick, add a check that ensures that the correct channel is dropped. So channel 1 is still in the channel list and channel2 is dropped.